collectors/ciscoduo/utils.js

```diff
@@ -39,7 +39,10 @@ function getAPILogs(client, objectDetails, state, accumulator, maxPagesPerInvoca
                                 getData();
                             }
                             else {
-                                nextPage = objectDetails.query.mintime;
+                                // If there is no data check if min time stamp is less than 1 hr move by 1 sec else set the last hour time stamp as nextPage
+                                const lastHourMoment = moment().subtract(1, 'hours').unix();
+                                const mintime = parseInt(objectDetails.query.mintime);
+                                nextPage = lastHourMoment > mintime ? lastHourMoment : mintime + 1;
```

Comment:
let's use `Math.min(mintime+1, lastHourMoment)` here

Reply:
Used Math.max(mintime+1, lastHourMoment) to get  latest time.